Gemfile

```diff
@@ -5,6 +5,11 @@ gem "activesupport", "< 4.0.0", :group => :compat_testing, :platform => "ruby"
 
 gem 'chef-config', path: "chef-config"
 
+# REMOVEME once there is a release of Ohai with these changes, and the
+# chef.gemspec is updated.
+gem 'ohai', github: 'chef/ohai',
```

Comment:
Just for my own sake, I'm marking locations that would need to be changed before commit.